.github/workflows/publish-sdk.yml

```diff
@@ -0,0 +1,44 @@
+name: Tonomy SDK publish
+
+on:
+  push:
+
+permissions:
+  contents: write
+
+  # pull_request:
+  #   types:
+  #     - closed
+  #   branches:
+  #     - development
+
+jobs:
+
+  publish-npm:
+    runs-on: ubuntu-20.04    
+    steps:
+      - name: 🏗 Setup repo
+        uses: actions/checkout@v3
+
+      
+      - name: 📦 Install
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.4.1
+          registry-url: https://registry.npmjs.org/
+
+      - name: "Automated Version Bump"
+        uses: "phips28/gh-action-bump-version@master"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          version-type:  'minor'
+        
+
+      - name: 🚀 Build and publish SDK
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.4.1
+      - run: npm publish
```

Comment:
don't you need to run `npm install` and `npm build` (or equivalent) before we publish? otherwise whoever installs it doesnt have a built library...

Reply:
@theblockstalk  I have not used npm install or build

Reply:
you need to. when you install it from npm, the module it uses is here in the `./dist` directory which you need to run `npm run build` to get
https://github.com/Tonomy-Foundation/Tonomy-ID-SDK/blob/development/package.json#L33